src/stories/Collapsible.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from 'styled-components'
+import { Meta, StoryObj } from '@storybook/react'
+import { useArgs } from '@storybook/preview-api'
+
+import { BorderRadius, Colors, Spacing } from '../tokens'
+import * as Collapsible from '../components/Collapsible'
+
+//* DECORATORS */
+
+const StoryPlaceholder = styled.div`
+  width: 600px;
+  padding: ${Spacing['5']};
+  border-radius: ${BorderRadius['3']};
+  border: 1px solid ${Colors.light.neutral.neutral200};
+`
+const StyledSpan = styled.header`
+  color: ${Colors.light.neutral.neutral400};
```

Comment:
the theme must accept dark mode

Reply:
This is only used for presentation, it isn't part of the component itself.